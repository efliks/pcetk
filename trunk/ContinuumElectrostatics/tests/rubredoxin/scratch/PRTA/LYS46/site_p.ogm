ON_GEOM_CENT 121 2.000000
(8.483210 32.509401 1.934769) 101 1.000000
(8.483210 32.509401 1.934769) 101 0.500000
(8.483210 32.509401 1.934769) 101 0.250000
