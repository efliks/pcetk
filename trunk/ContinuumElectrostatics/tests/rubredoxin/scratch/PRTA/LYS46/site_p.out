Starting /home/mikolaj/local/bin//my_3diel_solver for molecule named scratch/PRTA/LYS46//site_p
using the following physical conditions:
Exterior dielectric constant, epsext = 80
Solvent probe radius, solrad = 1.4
Ion exclusion layer thickness, sterln = 2
Temperature, T = 300
Ionic strength, ionicstr = 0.1
Hueckel factor (epsext*kappasq) = 0.843038
Debye length 1/kappa = 9.7414
ln(10) * kBolt * T = 0.00413389
kBolt = 5.98442e-06
conconv = 0.000602214
econv = 332.063
Bohr radius = 0.529177
Proton Charge = 1
elementary_charge = 1.60218e-19
energy_unit = 2.30707e-18
Avogadro_Number = 6.02214e+23
voltconv = 14.3996
pkconv = 241.903
  Solute interior dielectric, epsin1 = 1
  Protein interior dielectric constant, epsin2 = 4
Vacuum dielectric constant, epsvac = 1

No blab level set (so no blabbing)
Using finite difference method with lattice levels:
                   Centering Style   grid dimension    spacing
Coarsest level: ON_GEOM_CENT resolved to (13.319, 37.4012, 4.51304)) 121 2
Finer level:    (8.48321, 32.5094, 1.93477) 101 1
Finer level:    (8.48321, 32.5094, 1.93477) 101 0.5
Finest level:   (8.48321, 32.5094, 1.93477) 101 0.25


Self energy of scratch/PRTA/LYS46//site_p in three dielectric environment = 101.107
(probably in kcal/mole, INCLUDING GRID ARTEFACTS)


Interaction energy of scratch/PRTA/LYS46//site_p with background charge set in three dielectric environment = -5.40357
(probably in kcal/mole)



Interaction energies of scratch/PRTA/LYS46//site_p with charges of given instances of sites (probably in kcal/mole).
Format: site instance energy

0   0 : 0.0376586
0   1 : 8.98776e-05
1   0 : -0.00967504
1   1 : -0.319963
2   0 : 0.145913
2   1 : 0.000919457
3   0 : -0.0109659
3   1 : -0.0909345
4   0 : -0.034334
4   1 : -0.2262
5   0 : -0.0480933
5   1 : -1.1415
6   0 : 0.00266149
6   1 : -0.0911907
7   0 : 0.0542555
7   1 : -0.443127
8   0 : 0.00238127
8   1 : -0.046121
9   0 : 0.0417722
9   1 : -0.203906
10   0 : 0.00182266
10   1 : -0.0236471
11   0 : 0.117355
11   1 : 0.00156698
12   0 : 0.0664232
12   1 : -0.611594
13   0 : 0.0247716
13   1 : -0.256688
14   0 : -0.0746072
14   1 : -0.8372
15   0 : -0.00315398
15   1 : -0.224514
16   0 : 0.000642754
16   1 : -0.360168
17   0 : -0.0147589
17   1 : -0.116683
18   0 : 202.3
18   1 : 49.6959
19   0 : 0.0088833
19   1 : -0.146148
20   0 : 0.0134551
20   1 : -0.0853311
21   0 : 0.175484
21   1 : 0.00148155
22   0 : 0.00125777
22   1 : -0.101077
23   0 : -0.000548341
23   1 : -0.0337421
Total runtime: 14.74 s user 0.16 s system 
